Added CODE_OF_CONDUCT.md (#93)
AntonOsika/gpt-engineer: CODE_OF_CONDUCT.md

# Contributor Covenant Code of Conduct

## Our Pledge

We as members, contributors, and leaders pledge to make participation in our
community a harassment-free experience for everyone, regardless of age, body
size, visible or invisible disability, ethnicity, sex characteristics, gender

> identity and expression, level of experience, education, socio-economic status,

Reviewer: ```suggestion
identity or expression, level of experience, education, socio-economic status,
```

  any kind
* Trolling, insulting or derogatory comments, and personal or political attacks
* Public or private harassment
* Publishing others' private information, such as a physical or email address,
  without their explicit permission
* Other conduct which could reasonably be considered inappropriate in a
  professional setting

## Enforcement Responsibilities

Community leaders are responsible for clarifying and enforcing our standards of
acceptable behavior and will take appropriate and fair corrective action in
response to any behavior that they deem inappropriate, threatening, offensive,
or harmful.

Community leaders have the right and responsibility to remove, edit, or reject
comments, commits, code, wiki edits, issues, and other contributions that are
not aligned to this Code of Conduct, and will communicate reasons for moderation
decisions when appropriate.

## Scope

This Code of Conduct applies within all community spaces, and also applies when
an individual is officially representing the community in public spaces.
Examples of representing our community include using an official e-mail address,

> posting via an official social media account, or acting as an appointed

Reviewer: ```suggestion
posting using an official social media account, or acting as an appointed
```

representative at an online or offline event.

## Enforcement

Instances of abusive, harassing, or otherwise unacceptable behavior may be
reported to the community leaders responsible for enforcement at
anton.osika@gmail.com.
All complaints will be reviewed and investigated promptly and fairly.

> All community leaders are obligated to respect the privacy and security of the
> reporter of any incident.

Reviewer: ```suggestion
All community leaders are obligated to respect the privacy and security of reporters of incidents.
```
